### PR TITLE
WIP Scrape house hearings

### PR DESCRIPTION
**Pull request by crawfoal**
Functionality not yet implemented:

- pull in event ID and handle updates
- formatting of locations not at capitol complex
- proper handling of event times for events that are not in the Eastern Time Zone
- scraping of event committees and building that association

Let me know what else you find 🙂 Also, I'm not sure whether the above things should all be done in this PR or whether I should break it up. Keeping them, it could get kinda big and take a while, but it doesn't feel complete without them.

### CODE REVIEW

File: app/models/event.rb
Comment (jaydorsey) [inline app/models/event.rb:45]: If the only other edge case we've seen so far is presence of `capitol-complex` field, or presence of `field`, we can probably deal with these as 2 separate edge cases rather than rescuing the `NoMethodError`. That would help communicate that there are only (we think) 2 edge cases/formats.

These are the only 2 scenarios I've seen so far on the scrapes I've run but I will defer to you on this because I've only looked at a couple scrapes.

File: app/models/event.rb
Comment (jaydorsey) [inline app/models/event.rb:44]: Later, do you think it would be useful for us to build a look up table/method for these office locations so when we generate ICS files or sync to Google calendar, we can provide location information? I'm guessing most people in the area know these locations but once you include real physical location info you get distance and departure notifications for free on some phones/clients.
 
For example, RHOB is the Rayburn House Office Building, 45 Independence Ave SW, Washington, DC 20515.

If so, this might be a good secondary issue to create

File: db/migrate/20180610121037_add_raw_location_to_event.rb
Comment (jaydorsey) [inline db/migrate/20180610121037_add_raw_location_to_event.rb:3]: Great idea, saving this info while we're parsing!

File: app/lib/house_events_scraper.rb
Comment (jaydorsey) [inline app/lib/house_events_scraper.rb:11]: Later, we could also namespace these and split out into separate class files (`HouseEvents::Scraper`, `HouseEvents::Parser`, `HouseEvents::Sync`, etc) but we still have some time to think about that

**Review by jaydorsey — APPROVED** [review]
👍 

This looks great to me. Building this out in ruby was a great choice for grabbing the calendar only data.

If you can fix the travis/rubocop errors before you merge, we'll be set! Feel free to merge this after you update, or ping me if you want me to take another look.

Also, great job on including tests again!

**Comment by compostbrain** [comment]
@crawfoal Wanted to check in and see if you have time to get your ci checks passing so we can merge this pr?  Also, I noticed propublica got back to you and said they have fixed their api.  Maybe we can go back to the api approach?

**Pull request merged**
